ndarray-rand/src/lib.rs

```diff
@@ -29,11 +29,12 @@
 //! that the items are not compatible (e.g. that a type doesn't implement a
 //! necessary trait).
 
-use crate::rand::distributions::Distribution;
+use crate::rand::distributions::{Distribution, Uniform};
 use crate::rand::rngs::SmallRng;
+use crate::rand::seq::index::sample;
```

Comment:
I'd shorten this to `use crate::rand::seq::index`, and in the code use `index::sample`. Just seeing `sample` in the code is somewhat confusing because it's not clear what "sample" means.